### ISSUE TITLE
Npm package id distributed with different license

### ISSUE THREAD

**Issue by klimashkin** [cloudflare/a11y-focus-scope#1]
Hi! License in package differs from what [I see in this repo](https://github.com/cloudflare/a11y-focus-scope/blob/master/LICENSE)

What comes within npm package:

> Copyright (c) 2015 James Kyle <me@thejameskyle.com>
> Permission to use, copy, modify, and/or distribute this software for any purpose with or without fee is hereby granted, provided that the above copyright notice and this permission notice appear in all copies.
THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.

So it's not clear what license should be used.

**Comment by jamiebuilds** [comment]
The published version must have the old MIT version, if a new version gets published it will turn into a BSD license. Both licenses are okay for open source projects, I wouldn't worry about it.

**Comment by klimashkin** [comment]
Yeah, but I use this package in non open source software, and I have to include all licenses that we use in it. So that why I was wondering :)
